src/components/Synthetics/GmSwap/GmSwapBox/Swap.tsx

```diff
@@ -4,11 +4,11 @@ export function Swap() {
   return (
     <div className="z-2 relative">
       <div
-        className="absolute -top-25 left-1/2 ml-[-17.825px]
-                   flex size-35 cursor-not-allowed select-none items-center justify-center rounded-31
-                  bg-blue-600 text-white opacity-80"
+        className="absolute -top-25 left-1/2 ml-[-17.825px] flex
+                   size-35 cursor-not-allowed select-none items-center justify-center rounded-31 bg-[#1b1e32]
```

Comment:
1b1e32 looks similar to colors we have, is it different ? if its niche then ok

Reply:
It's exceptional color, so we decide to hardcode it